homeassistant/components/voip/assist_satellite.py

```diff
@@ -193,6 +194,11 @@ async def _do_announce(
 
         Optionally run a voice pipeline after the announcement has finished.
         """
+        if announcement.media_id_source != "tts":
+            raise HomeAssistantError(
+                "VoIP does not currently support non-TTS announcements"
```

Comment:
As this is a service action... should we add translation support for this from the get-go?

../Frenck